theme/skin/rexify.org/_navigation.html.ep

```diff
@@ -1,23 +1,23 @@
-% my $nav = $site->{data}{nav};
+% my $nav = $site->{data}->{nav};
```

Comment:
Nice catch! :)

Also, I believe this would become `my $nav = $site->nav;` with the built-in nav structure.

Reply:
I'll send a follow-up PR when I find out what the problem is with the validation of the `nav` site attribute. For now, I didn't get it to work due to the `href` error mentioned above.